.gitignore

```diff
@@ -11,4 +12,6 @@
 *.ntvs*
 *.njsproj
 *.sln
-*.sw?
\ No newline at end of file
+*.sw?
+
+yarn-error.log
```

Comment:
I recommend you insert a blank line at the end of each file.
It makes a difference in git more clean.